src/main/java/org/springframework/samples/petclinic/rest/controller/OwnerRestController.java

```diff
@@ -117,9 +122,9 @@ public OwnerRestController(ClinicService clinicService,
     }
 
     @PreAuthorize("hasRole(@roles.OWNER_ADMIN)")
-    @RequestMapping(value = "/owners/{ownerId}", method = RequestMethod.DELETE, produces = "application/json")
     @Transactional
-    public ResponseEntity<Void> deleteOwner(@PathVariable("ownerId") int ownerId) {
+    @Override
+    public ResponseEntity<OwnerDto> deleteOwner(@Min(0) @ApiParam(value = "The ID of the owner.", required = true) @PathVariable("ownerId") Integer ownerId) {
```

Comment:
I think we don't have to declare a second time all annotations generated in the implemented interface (Min, ApiParam, PathVariable). It could differ.
This comment is valid for all overriden methods.

Reply:
Fixed